src/app/content/utils/idUtils.ts

```diff
@@ -1,2 +1,3 @@
 export const stripIdVersion = (id: string): string => id.split('@')[0];
 export const getIdVersion = (id: string): string | undefined => id.split('@')[1];
+export const stripArchiveVersion = (id: string): string => id.split('__')[1];
```

Comment:
i think the underscore formatting is search specific, in archive its still formatted `id@version`

Reply:
since this formatting is specific to search, i think it would make more sense to leave it embedded in that function that is like `getIndexInfo` or whatever